modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -143,22 +153,22 @@ This command creates a YAML file for each `CredentialsRequest` object.
 [source,terminal]
 ----
 $ ccoctl ibmcloud create-service-id \
-  --credentials-requests-dir=<path_to_credential_requests_directory> \// <1>
-  --name=<cluster_name> \// <2>
-  --output-dir=<installation_directory> \// <3>
-  --resource-group-name=<resource_group_name> <4>
-----
-<1> Specify the directory containing the files for the component `CredentialsRequest` objects.
-<2> Specify the name of the {product-title} cluster.
-<3> Optional: Specify the directory in which you want the `ccoctl` utility to create objects. By default, the utility creates objects in the directory in which the commands are run.
-<4> Optional: Specify the name of the resource group used for scoping the access policies.
+  --credentials-requests-dir=<path_to_credential_requests_directory> \
+  --name=<cluster_name> \
+  --output-dir=<installation_directory> \
+  --resource-group-name=<resource_group_name>
+----
+* `<path_to_credential_requests_directory>`: Specify the directory containing the files for the `CredentialsRequest` objects.
```

Comment:
we were talking about the accessibility of this format in the tiger team and I think it is preferred to have some sort of sentence offset a list like this; there is no formal guidance, but if you think of a screen reader, it makes sense; I used "Additional information about <xyz> parameters:" and connected it with a `+`

something to consider